src/misc/L10N.ts

```diff
@@ -8,6 +8,19 @@ const singularRegex = /<sn>(((?!<\/sn>).)*)<\/sn>/g;
 export var L10N = {
   format: (key: string, ...args: any[]) => {
     let value = key.toLocaleString();
+    // Try to find a soft match
+    if (value == key) {
```

Comment:
4 `if` imbricked like this is pretty hard to read.

You can simply align them flat.

    const originalValue = key;
    let possiblyTranslated = originalValue.toLocaleString();

    if(possiblyTranslated == originalValue) {
        possiblyTranslated = originalValue.ToUpperCase().toLocaleString();
    }

    if(possiblyTranslated == originalValue) {
         // do lower case
    }

    if(possiblyTranslated == originalValue) {
        // do first char upper case
    }


etc...